lib/swiftype-app-search/exceptions.rb

```diff
@@ -3,7 +3,7 @@ class ClientException < StandardError
     attr_reader :errors
 
     def initialize(response)
-      @errors = response['errors'] || [ response ]
+      @errors = response['errors'] || [response]
```

Comment:
General curiosity - in what circumstances was the `|| [response]` required? 
Was it originally added to handle the `InvalidDocument` error codes?

Reply:
Hi @kurtfunai, thanks for these submissions! I think the intention here is a graceful fallback in the case that the Swiftype API doesn't return an `'errors'` key in the json response.